lib/kernel/src/logger_formatter.erl

```diff
@@ -35,7 +35,7 @@
                     template        => template(),
                     time_designator => byte(),
                     time_offset     => integer() | [byte()]}.
--type template() :: [metakey() | {metakey(),template(),template()} | string()].
```

Comment:
The type should be:

    -type template() :: [metakey() | {metakey(),template(),template()} | unicode:chardata()].